modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+
+* Before this update, when you used {tekton-results} to store pod logs from pipelines and tasks, the operation to store the logs sometimes failed. The logs would include the `UpdateLog` action failing with the `canceled context` error. With this update, the operation completes correctly. 
+
+* Before this update, when you passed a parameter value to a pipeline or task and this value included more than one variable, and they use both full and short reference formats, for example, `$(tasks.task-name.results.variable1) + $(variable2)`, {pipelines-shortname} did not interpret the value correctly; the pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
+
+* Before this update, {tekton-chains} failed to generate correct attestations when a task run included multiple tasks with the same name. For instance, when using a matrix of tasks, the attestation was generated for the first image. With this update, {tekton-chains} generates attestations for all tasks within the task run, ensuring complete coverage.
+
+* Before this update, when you used the `skopeo-copy` task defined in the {pipelines-shortname} installation namespace and set its `VERBOSE` parameter to `false`, the task failed. With this update, the task completes normally.
+
+* Before this update, when using {pac}, if you set the `concurrency_limit` spec in the global `Repository` CR named `pipelines-as-code` in the `openshift-pipelines` or `pipelines-as-code` namespace, which provides default settings for all `Repository` CRs, the {pac} watcher crashed. With this update, the {pac} watcher operates correctly with this setting.
+
+* Before this update, all tasks in {pipelines-shortname} included an extra step compared to the cluster tasks of the same name that were available in previous versions of {pipelines-shortname}. This extra step increased the load on the cluster. With this update, the tasks no longer include the extra step as it is integrated into the first step.
+
+* Before this update, when you used one of the `s2i-*` tasks defined in the {pipelines-shortname} installation namespace and set its `CONTEXT` parameter, the parameter was not interpreted correctly and the task failed. With this update, the task interprets the `CONTEX`T parameter correctly and completes successfully.
+
+* Before this update, in {tekton-chains} the in-toto provenance metadata, `URI` and `Digest`, were incomplete. They contained only the information of remote `Pipeline` and `Task` resources, but were missing the information of the remote `StepAction` resource. With this update, the provenance of the remote `StepAction` resource is recorded in the task run status and inserted into the in-toto provenance, which results in complete in-toto provenance metadata.
```

Comment:
```suggestion
* Before this update, in {tekton-chains} the in-toto provenance metadata, `URI` and `Digest` values?, were incomplete. The values contained only the information of remote `Pipeline` and `Task` resources, but were missing the information of the remote `StepAction` resource. With this update, the provenance of the remote `StepAction` resource is recorded in the task run status and inserted into the in-toto provenance, which results in complete in-toto provenance metadata.
```

Reply:
I'm wondering if there can be something more specific than "they" to reference `URI` and `Digest`. Like values?

Reply:
"Values" should work, yes.